src/nih_wayfinding/app/scripts/routing/directions-service.js

```diff
@@ -130,6 +132,38 @@
         }
 
         /**
+         * Get a summary for a route, including time, distance and # of turns
```

Comment:
Right. Although, the geojson format we set up doesn't really allow for a trip level properties object. Let me see if I can reshuffle things a bit, maybe save the itinerary summary to a private var in Directions.